packages/bpk-component-datepicker/stories.js

```diff
@@ -57,15 +57,21 @@ class CalendarContainer extends Component {
   }
   render() {
     return (
-      <BpkDatepicker
-        {...this.props}
-        date={this.state.date}
-        onDateSelect={(date) => {
-          this.setState({ date });
-          action('Selected date')(date);
-        }}
-        onMonthChange={action('Changed month')}
-      />
+      <div id="datepicker-element">
+        <div id="application-element">
+          <BpkDatepicker
+            {...this.props}
+            date={this.state.date}
+            onDateSelect={(date) => {
+              this.setState({ date });
+              action('Selected date')(date);
+            }}
+            onMonthChange={action('Changed month')}
+            getApplicationElement={() => document.getElementById('application-element')}
```

Comment:
nit: Shouldn't we prefer the ref method instead?

Reply:
Could do, but a lot of usage in Skyscanner (particularly with OC components) depends on containers defined outwith the components react tree (e.g. the header culture selector needs to get application element defined in microsite, not in header).

Reply:
Good point, but shouldn't we use the React-way when we can?

Reply:
Myself and hugo caught up offline and discussed that the most common usage in Skyscanner is preferred.